### ISSUE TITLE
New Font Needed for Nepali variation of Devanagari with Altered Shape for झ and related conjuncts

### ISSUE THREAD

**Issue by srijon2024** [TiroTypeworks/Indigo#59]
A Font would be required to replace standard Hindi झ with standard Nepali झ (different shape). Modern Nepali Preeti-based Fonts do help but the Dotted consonants used in Hindi like क़, ख़, ग़, ज़, झ़, ड़, ढ़, फ़ don't exist in them.

We need a Tiro Devanagari Nepali font with everything being same as Tiro Devanagari Hindi but replacement of shape of झ with Nepali style.

This font would be specifically used for Hindi, Nepali, Bhojpuri, Maithili, Awadhi, Rajbongshi/Kamtapuri, Kumaoni, Garhwali and Doteli. It will serve users in India, Nepal and Bangladesh.

[Tiro Devanagari Nepali.pdf](https://github.com/user-attachments/files/16310524/Tiro.Devanagari.Nepali.pdf)

**Comment by tiroj** [comment]
Are you aware of any additional conjunct forms that would be useful for Nepali, beyond the झ conjuncts shown in the PDF? I do not have good conjunct analysis data for Nepali.

Am I correct in remembering that Nepali does not use the anusvara dot for nasalisation, but alwayys the candrabindu form?

**Comment by srijon2024** [comment]
> Are you aware of any additional conjunct forms that would be useful for Nepali, beyond the झ conjuncts shown in the PDF? I do not have good conjunct analysis data for Nepali.
> 
> Am I correct in remembering that Nepali does not use the anusvara dot for nasalisation, but alwayys the candrabindu form?

No, rest all conjunct consonants are same. Only झ is different.

In a Nepali font you should keep the traditional forms of क्त and क्र  instead of the simplistic क्‍त and क्र. Nepali additionally uses र् + ZWJ + य = र्‍य.

In general Nepali doesn't use dotted consonants which Hindi has, but as a unicode font they must be retained as they are used in some languages of Nepali territories bordering India. 

![image](https://github.com/user-attachments/assets/144e2c0e-d24e-4dfd-90a0-c8857b1805c7)

Newari and occasionally Nepali use another form of झ which is seen in some Brands (Sajha Publication, Sajha Travels).

![image](https://github.com/user-attachments/assets/8d6e277b-e415-4f8a-9d4e-350301f32276)
![image](https://github.com/user-attachments/assets/7b39f099-f173-4ccf-bf44-94a2d273cec1)
![image](https://github.com/user-attachments/assets/fd50e4af-bb22-4664-929d-912684cddb8a)

**Comment by srijon2024** [comment]
> Am I correct in remembering that Nepali does not use the anusvara dot for nasalisation, but alwayys the candrabindu form?

Yes, Nepali uses Chandrabindu everywhere. The positioning of Chandrabindu with clear visibility has remained a big issue with Unicode fonts for Devanagari.

But Bindu (Anunasika) is retained for Sanskrit words.

Nepali uses a lot of conjuncts, more often than Hindi, but the pattern in both remains almost same. The differences have been shown in the previous reply.
